dist/src/main/java/io/camunda/webapps/controllers/BackupController.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.controllers;
+
+import io.camunda.management.backups.Error;
+import io.camunda.management.backups.HistoryBackupDetail;
+import io.camunda.management.backups.HistoryBackupInfo;
+import io.camunda.management.backups.HistoryStateCode;
+import io.camunda.management.backups.TakeBackupHistoryResponse;
+import io.camunda.webapps.backup.BackupException;
+import io.camunda.webapps.backup.BackupException.*;
+import io.camunda.webapps.backup.BackupService;
+import io.camunda.webapps.backup.BackupStateDto;
+import io.camunda.webapps.backup.GetBackupStateResponseDetailDto;
+import io.camunda.webapps.backup.GetBackupStateResponseDto;
+import io.camunda.webapps.backup.TakeBackupRequestDto;
+import io.camunda.webapps.backup.repository.BackupRepositoryProps;
+import io.camunda.webapps.profiles.ProfileOperateTasklistStandalone;
+import io.camunda.zeebe.util.VisibleForTesting;
+import io.micrometer.common.lang.NonNull;
+import java.math.BigDecimal;
+import java.util.Arrays;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.endpoint.annotation.DeleteOperation;
+import org.springframework.boot.actuate.endpoint.annotation.ReadOperation;
+import org.springframework.boot.actuate.endpoint.annotation.Selector;
+import org.springframework.boot.actuate.endpoint.annotation.WriteOperation;
+import org.springframework.boot.actuate.endpoint.web.WebEndpointResponse;
+import org.springframework.boot.actuate.endpoint.web.annotation.WebEndpoint;
+import org.springframework.stereotype.Component;
+
+@Component
+@WebEndpoint(id = "backup-history")
+@ProfileOperateTasklistStandalone
```

Comment:
This must still be standalone until we have completely unified the implementations

Reply:
Just spring startup issues, I'll be removing the standalone very soon when I'll be removing all the implementations of BackupServices from tasklist and move all the startup code to `dist`

Reply:
Alright, no need to block for this, but I'm still curious to understand it :smile: